### PR TITLE
[IN-2011] fixed core/cask/cask-versions

### PR DESCRIPTION
**Pull request by lmesz-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by MrBlaise — APPROVED** [review]

**Review by fehersanyi-bitrise — DISMISSED** [review]

**Review by MrBlaise — DISMISSED** [review]

File: roles/brew-repos-fix/files/setup_brew_core.sh [hunk 1]
```diff
@@ -8,8 +8,11 @@ brew_repository=$(brew --repository)
 
 rm -rf "$brew_repository/Library/Taps/homebrew"
 
+rm -rf ~/mirrors/github.com/bitrise-io/homebrew-core
+
 mkdir -p ~/mirrors/github.com/bitrise-io/
 
+
```
Comment (fehersanyi-bitrise) [inline roles/brew-repos-fix/files/setup_brew_core.sh:15]: why is this here?

**Review by fehersanyi-bitrise — DISMISSED** [review]

File: roles/brew-repos-fix/files/setup_brew_core.sh (hunk 1 above)
Comment (lmesz) [reply roles/brew-repos-fix/files/setup_brew_core.sh:15]: To mirror the directory structure of github

Comment (fehersanyi-bitrise) [reply roles/brew-repos-fix/files/setup_brew_core.sh:15]: sorry, what I meant is the random new line :D

File: roles/certs/tasks/main.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@
     checksum: md5:08a45128fa238443623421dd2c9887ab
 
 - name: Add Apple certificate
-  shell: bash -l -c "sudo security add-certificates -k /Library/Keychains/System.keychain $HOME/apple_cert.cer"
+  shell: bash -l -c "sudo security add-certificates -k /Library/Keychains/System.keychain $HOME/apple_cert.cer || true"
```
Comment (bitrise-tom) [inline roles/certs/tasks/main.yml:9]: How many errors is this going to mask and what are the implications?

Comment (lmesz-bitrise) [reply roles/certs/tasks/main.yml:9]: I hope only the already added. There will be a permanent fix in this PR. https://github.com/bitrise-io/osx-box-bootstrap/pull/329

File: roles/brew-repos-fix/files/setup_brew_core.sh (hunk 1 above)
Comment (lmesz-bitrise) [reply roles/brew-repos-fix/files/setup_brew_core.sh]: Upsz, right, removed.

**Review by MrBlaise — DISMISSED** [review]

**Review by bitrise-tom — DISMISSED** [review]

File: roles/brew-repos-fix/files/setup_brew_core.sh (hunk 1 above)
Comment (lmesz-bitrise) [reply roles/brew-repos-fix/files/setup_brew_core.sh]: I already fixed it. Can you @bitrise-tom help me with this?

**Review by peterheja-bitrise — APPROVED** [review]
Approved, but let's record this as a tech debt.

**Comment by lmesz-bitrise** [comment]
> Approved, but let's record this as a tech debt.

Which part?

**Comment by peterheja-bitrise** [comment]
> > Approved, but let's record this as a tech debt.
> 
> Which part?

`rm -rf`-ing. This is a hack.

**Review by MrBlaise — APPROVED** [review]
